content/articles/secondary-dns-provider-easy-dns.md

```diff
@@ -44,19 +44,15 @@ From the *DNS settings* tab, you'll tell EasyDNS the primary name server. In thi
 
 ![DNSimple configuration is successful](/files/easy-dns-domain-admin.png)
 
-Now add the IP address you configured at DNSimple: `50.31.225.92`
+Now add the DNSimple primary server IP address `18.189.127.127` and confirm the changes.
 
 <info>
   At the time of writing, **EasyDNS** only accepts IPv4 for Secondary DNS
 </info>
 
 ![DNSimple configuration is successful](/files/easy-dns-adding-dnsimple-ip.png)
 
-Then confirm the changes.
-
-![DNSimple configuration is successful](/files/easy-dns-confirm.png)
-
-It can take up to one hour for the first zone transfer. To check if your secondary is configured correctly, select the *Tools* tab, then *check* the zone file.
+It can take up to one hour for the first zone transfer. To check if your secondary is configured correctly, select the *DNS Settings* tab, then *check* the zone cache. The link shows you what EasyDNS is currently storing in its cache as your zone information. This should match what you see in your DNS settings within the primary name server at DNSimple.
```

Comment:
> then *check* the zone cache

there's no box available for this to be an action, I'm also not clear on why it's italicized — is this supposed to say click with zone cache italicized?

Reply:
I have updated the text to clarify clicking the 'zone cache' link.